src/ukify/ukify.py

```diff
@@ -449,7 +449,26 @@ class PeError(Exception):
 
 def pe_add_sections(uki: UKI, output: str):
     pe = pefile.PE(uki.executable, fast_load=True)
-    assert pe.FILE_HEADER.PointerToSymbolTable != 0 or len(pe.__data__) % pe.OPTIONAL_HEADER.FileAlignment == 0
+
+    # Old stubs might have been stripped, leading to unaligned raw data values, so let's fix them up here.
+    for i in range(len(pe.sections)):
+        oldp = pe.sections[i].PointerToRawData
+        oldsz = pe.sections[i].SizeOfRawData
+        pe.sections[i].PointerToRawData = round_up(pe.sections[i].PointerToRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        pe.sections[i].SizeOfRawData = round_up(pe.sections[i].SizeOfRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        padp = pe.sections[i].PointerToRawData - oldp
+        padsz = pe.sections[i].SizeOfRawData - oldsz
+
+        for j in range(i + 1, len(pe.sections)):
+            pe.sections[j].PointerToRawData += padp + padsz
```

Comment:
Possibly less indexes to throw around
```suggestion
    for i, section in enumerate(pe.sections):
        oldp = sections.PointerToRawData
        oldsz = sections.SizeOfRawData
        section.PointerToRawData = round_up(oldp, pe.OPTIONAL_HEADER.FileAlignment)
        sections.SizeOfRawData = round_up(oldsz, pe.OPTIONAL_HEADER.FileAlignment)
        padp = section.PointerToRawData - oldp
        padsz = section.SizeOfRawData - oldsz

        for later_section in pe.sections[i+1:]:
            later_section.PointerToRawData += padp + padsz
```